README.md

```diff
@@ -37,8 +37,14 @@ All commands are available in the Command Palette and Context Menus for the Expl
 ext install adhamu.history-in-sublime-merge
 ```
 
-## Requirements
+## Requirements for Windows
```

Comment:
I think this can be removed. These requirements are accurate for other operating systems

Reply:
No worries, done :+1: